devlab/tests/test_verify_dst_net_segm_id.py

```diff
@@ -0,0 +1,54 @@
+import tests.functional_test as functional_test
+
+TEST_TENANT_NAME = 'tenant4'
+TEST_TENANT_NETWORKS = ['tenantnet4_segm_id_cidr1',
+                        'tenantnet4_segm_id_cidr2']
+
+
+class VerifyDstCloudFunctionality(functional_test.FunctionalTest):
+
+    def setUp(self):
+        # take all segmentation ids on source and destination
+        self.src_cloud.switch_user(user=self.src_cloud.username,
+                                   password=self.src_cloud.password,
+                                   tenant=self.src_cloud.tenant)
+        self.src_test_tenant_name = \
+            self.src_cloud.get_tenant_id(TEST_TENANT_NAME)
+
+        self.src_all_nets_sids = []
+        for net in self.src_cloud.neutronclient.list_networks(
+                all_tenants=True)['networks']:
+            if net['provider:segmentation_id']:
+                self.src_all_nets_sids.append(
+                    (net['name'], net['provider:segmentation_id']))
+
+        self.dst_cloud.switch_user(user=self.dst_cloud.username,
+                                   password=self.dst_cloud.password,
+                                   tenant=self.dst_cloud.tenant)
+        self.dst_test_tenant_name = \
+            self.dst_cloud.get_tenant_id(TEST_TENANT_NAME)
+
+        self.dst_net_sid_mapping = []
+        self.dst_all_nets_sids = []
+        for net in self.dst_cloud.neutronclient.list_networks(
+                all_tenants=True)['networks']:
+            if net['tenant_id'] == self.dst_test_tenant_name\
+                    and net['name'] in TEST_TENANT_NETWORKS:
+                self.dst_net_sid_mapping.append(
+                    (net['name'], net['provider:segmentation_id']))
+                continue
+            elif net['provider:segmentation_id']:
+                self.dst_all_nets_sids.append(
+                    (net['name'], net['provider:segmentation_id']))
+
+    def test_check_segm_id_changed_on_dst(self):
```

Comment:
It is requirement to check if network migrated to destination with new segmentation ID which does not exist in source and destination clouds